PurchasesTests/TestHelpers/SnapshotTesting+Extensions.swift

```diff
@@ -37,12 +37,22 @@ private extension Encodable {
     }
 
     func asFormattedData() throws -> Data {
+        var formatting: JSONEncoder.OutputFormatting {
+            var result: JSONEncoder.OutputFormatting = [
+                .prettyPrinted,
+                .sortedKeys
+            ]
+
+            if #available(iOS 13.0, tvOS 13.0, *) {
+                result.insert(.withoutEscapingSlashes)
```

Comment:
does this mean we get a different result when running on iOS 12 devices?

Reply:
Oh yeah, I think this is gonna fail on `main` when we run tests on iOS 12.
I guess we can remove this. It just means that URLs will be encoded in JSON like this: `"https:\/\/api.revenuecat.com\/v1\/subscribers\/user\/intro_eligibility"`

Reply:
agreed, I think it's better to have backwards compatibility 👍